mixins/demo/localize-mixin.html

```diff
@@ -6,21 +6,16 @@
 		<link rel="stylesheet" href="../../components/demo/styles.css" type="text/css">
 		<script type="module">
 			import '../../components/demo/demo-page.js';
-			import { html, LitElement } from 'lit';
-			import { LocalizeDynamicMixin } from '../../mixins/localize-dynamic-mixin.js';
-			window.html = html;
-			window.LitElement = LitElement;
-			window.LocalizeDynamicMixin = LocalizeDynamicMixin;
 		</script>
 	</head>
 	<body unresolved>
 		<d2l-demo-page page-title="localize-mixin">
 
 			<h2>localize()</h2>
-
 			<d2l-demo-snippet>
-
 				<script type="module">
+import { html, LitElement } from 'lit';
+import { LocalizeDynamicMixin } from '../../mixins/localize-dynamic-mixin.js';
```

Comment:
I think this will show up converted to a full webserver path, which is a little odd, but not a huge problem.

Reply:
Correction: Just converts to a relative path into `node_modules`, which might be a bad example of how to import it, but still not a problem.